openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -515,6 +516,17 @@ def _inner(request):  # pylint: disable=missing-docstring
         else:
             response.content = msg
 
+        # Fire an event for login attempt
+        event_name = 'edx.user.attempt_login'
+        event_data = {
+            'email': request.POST.get('email'),
+            'remember': request.POST.get('remember'),
+            'username': request.user.username,
+            'user_id': request.user.id
+        }
+        event_data.update(response_dict)
```

Comment:
What data will `response_dict` contain?

Reply:
can you send a complete JSON of the entire event?

Reply:
In case of successful login attempt, `response_dict` contains `success(boolean)`, `redirect_url` and in case of failed attempt, `response_dict` contains `success(boolean)`, `value(str: reason of failure)`.

Reply:
JSON files have been uploaded.